nat-lab/tests/test_dns.py

```diff
@@ -867,6 +867,11 @@ async def test_dns_ttl_value() -> None:
 
         EXPECTED_TTL_VALUE = 1234
 
+        def features() -> Features:
+            features = test_features()
+            features.dns = FeatureDns(exit_dns=None, ttl_value=EXPECTED_TTL_VALUE)
+            return features
+
```

Comment:
In this specific case I think it was okay since the function was defined inside of the test and the only usage was a few lines later, but the function was unnecessary so I defined the features directly instead of wrapping them in a function